async/src/main/java/org/odk/collect/async/Scheduler.kt

```diff
@@ -81,7 +81,8 @@ interface Scheduler {
 
     enum class NetworkType {
         WIFI,
-        CELLULAR
+        CELLULAR,
+        OTHER
```

Comment:
`OTHER` would just be any other connected network. Android devices can theoretically have network delivered via Ethernet or Bluetooth for example. These kinds of connections would still satisfy WorkManager's (and our won) "connected" constraint.